lib/AppInfo/Application.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @author Joachim Bauch <mail@joachim-bauch.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Spreed\AppInfo;
+
+use OCP\AppFramework\App;
+use OCA\Spreed\Controller\PersonalSettingsController;
+
+class Application extends App {
+
+    public function __construct () {
+        parent::__construct('spreed');
+        $container = $this->getContainer();
+
+        $container->registerAlias('PersonalSettingsController', PersonalSettingsController::class);
```

Comment:
Hmm, if I remove it, loading the settings fails with this error (from `nextcloud.log`):
```
"Exception: {\"Exception\":\"OCP\\\\AppFramework\\\\QueryException\",\"Message\":\"Could not resolve PersonalSettingsController! Class PersonalSettingsController does not exist\",\"Code\":0,\"Trace\":\"#0 \\\/srv\\\/www\\\/lib\\\/private\\\/AppFramework\\\/Utility\\\/SimpleContainer.php(117): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->resolve('PersonalSetting...')\\n#1 \\\/srv\\\/www\\\/lib\\\/private\\\/AppFramework\\\/DependencyInjection\\\/DIContainer.php(540): OC\\\\AppFramework\\\\Utility\\\\SimpleContainer->query('PersonalSetting...')\\n#2 \\\/srv\\\/www\\\/apps\\\/spreed\\\/personal.php(24): OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer->query('PersonalSetting...')\\n#3 \\\/srv\\\/www\\\/lib\\\/private\\\/legacy\\\/app.php(757): include('\\\/srv\\\/www\\\/apps\\\/s...')\\n#4 \\\/srv\\\/www\\\/settings\\\/personal.php(184): OC_App::getForms('personal')\\n#5 \\\/srv\\\/www\\\/lib\\\/private\\\/Route\\\/Route.php(155) : runtime-created function(1): require_once('\\\/srv\\\/www\\\/settin...')\\n#6 [internal function]: __lambda_func()\\n#7 \\\/srv\\\/www\\\/lib\\\/private\\\/Route\\\/Router.php(298): call_user_func('\\\\x00lambda_603', Array)\\n#8 \\\/srv\\\/www\\\/lib\\\/base.php(1000): OC\\\\Route\\\\Router->match('\\\/settings\\\/perso...')\\n#9 \\\/srv\\\/www\\\/index.php(40): OC::handleRequest()\\n#10 {main}\",\"File\":\"\\\/srv\\\/www\\\/lib\\\/private\\\/AppFramework\\\/Utility\\\/SimpleContainer.php\",\"Line\":102}"
```